images/alpine/cloudbuild.yaml

```diff
@@ -1,19 +1,40 @@
 steps:
-  - name: gcr.io/cloud-builders/docker
-    args:
-    - build
-    - -t
-    - gcr.io/$PROJECT_ID/alpine:$_GIT_TAG
-    - --build-arg=IMAGE_ARG=gcr.io/$PROJECT_ID/alpine:$_GIT_TAG
-    - .
+# Set up multiarch support.
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['run', '--rm', '--privileged', 'multiarch/qemu-user-static', '--reset', '-p', 'yes']
+    id: 'qemu'
```

Comment:
The `id`s are not necessary (perhaps they are useful for debugging but we don't currently do anything with them.)

Reply:
'id' has been removed.